### ISSUE TITLE
Cancelled milking cow, give a ghost milk bucket

### ISSUE THREAD

**Issue by AriesStar1337** [KubeJS-Mods/KubeJS#581]
### Minecraft Version

1.19.2

### KubeJS Version

1902.6.0-build.142

### Rhino Version

1902.2.2-build.264

### Architectury Version

6.5.69

### Forge/Fabric Version

Forge 43.2.0

### Describe your issue

Right click a cow with a bucket cause a milking sound effect and a ghost milk bucket
Code:
```
ItemEvents.entityInteracted((event) => {
	event.cancel()
})
```

**Comment by TheLordofMelons** [comment]
Are you canceling it on the client as well?  If something is canceled on the server but not on the client it can cause a desync in certain situations.

**Comment by MaxNeedsSnacks** [comment]
Yeah, starting in 1.19, events are split between logical client and server as well, so you will need to resync the inventory or cancel the event on the client side as well. We're planning on adding startup events that fire for both logical client and server though.

**Comment by MaxNeedsSnacks** [comment]
We unfortunately don't have the time to support Minecraft versions older than 1.20 anymore, so I'll be closing this for now. You might still be able to find help on our Discord if you submit a ticket with the exact errors, and if it turns out this is a major KubeJS bug, we will of course go ahead and fix it still.
